Data/DbContext.cs

```diff
@@ -1,18 +1,21 @@
 using J3.Models;
 using Microsoft.EntityFrameworkCore;
+using J3.Models;
 
 namespace J3.Data
 {
-    public class ApplicationDbContext : DbContext
+    public class ColourContext : DbContext
     {
-        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
-            : base(options) { }
+        public ColourContext(DbContextOptions<ColourContext> options)
+            : base(options)
+        {
+        } //todo What's this?
```

Comment:
If it helps I can make another PR and add some comments to this file to make your life easier. Let me know

Reply:
All good, it was a note to self